.github/workflows/wasmBuild.yml

```diff
@@ -0,0 +1,17 @@
+name: WebAssmelby build
+on: [push, pull_request]
+jobs:
+  test:
+    name: BezierKit WASM support
+    runs-on: ubuntu-latest
+    container: ghcr.io/swiftwasm/carton:0.12.1
```

Comment:
any reason this uses `carton:0.12.1` whereas `travis/script.sh` uses `carton:latest` in the docker run command?